front/lib/models/assistant/actions/dust_app_run.ts

```diff
@@ -157,6 +158,10 @@ AgentDustAppRunAction.init(
       type: DataTypes.JSONB,
       allowNull: true,
     },
+    step: {
```

Comment:
Do we want to just set a default 0 for now and remove later ? easier than migrating no ?